subprojects/core/src/main/java/org/gradle/internal/progress/BuildProgressLogger.java

```diff
@@ -56,23 +58,36 @@ public void projectsLoaded(int totalProjects) {
     public void beforeEvaluate(String projectPath) {}
 
     public void afterEvaluate(String projectPath) {
-        if (!taskGraphPopulated) {
+        if (!rootTaskGraphPopulated) {
             buildProgress.progress("", false);
         }
     }
 
     public void graphPopulated(int totalTasks) {
-        taskGraphPopulated = true;
+        rootTaskGraphPopulated = true;
         buildProgress.completed();
         buildProgress = loggerProvider.start(EXECUTION_PHASE_DESCRIPTION, EXECUTION_PHASE_SHORT_DESCRIPTION, totalTasks);
     }
 
+    public void nestedTaskGraphPopulated(int totalTasks) {
```

Comment:
In my mind, "nested" encompasses builds that are triggered by the root build, for example: `buildSrc`, composite (included), and `GradleBuild` builds. I would consider "nested" a superset of just composite builds.